packages/cli/src/index.ts

```diff
@@ -3,12 +3,13 @@
 import chalk from 'chalk';
 import yargs from 'yargs-parser';
 import { createTutorial } from './commands/create/index.js';
+import { ejectRoutes } from './commands/eject/index.js';
 import { pkg } from './pkg.js';
 import { errorLabel, primaryLabel, printHelp } from './utils/messages.js';
 
-type CLICommand = 'version' | 'help' | 'create';
+type CLICommand = 'version' | 'help' | 'create' | 'eject';
 
-const supportedCommands = new Set(['version', 'help', 'create']);
+const supportedCommands = new Set<string>(['version', 'help', 'create', 'eject'] satisfies CLICommand[]);
```

Comment:
Can't we just type this as `Set<CLICommand>`?

Reply:
We can't because then we get a type error below when trying to do `supportedCommands.has(string)`. 

The reason I made that change was to make sure the values are checked against the `CLICommand` type.